From ace1351970901aa7228a744b37ca2176cb933f6e Mon Sep 17 00:00:00 2001
Message-ID: <ace1351970901aa7228a744b37ca2176cb933f6e.1701297273.git.sharlatanus@gmail.com>
In-Reply-To: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
References: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 29 Oct 2023 22:25:00 +0000
Subject: [PATCH v3 51/66] gnu: go-github-com-aws-aws-sdk-go-v2-service-sts:
 Move to (gnu packages golang-web).

* gnu/packages/golang.scm (go-github-com-aws-aws-sdk-go-v2-service-sts): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: I3503dbde93a46ff47dcc035243a7f320b7090794
---
 gnu/packages/golang-web.scm | 11 +++++++++++
 gnu/packages/golang.scm     | 10 ----------
 2 files changed, 11 insertions(+), 10 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 4bc9c944cb..aadf8ac73e 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -203,6 +203,17 @@ (define-public go-github-com-aws-aws-sdk-go-v2-service-ssooidc
     (propagated-inputs
      (list go-github-com-aws-smithy-go))))
 
+(define-public go-github-com-aws-aws-sdk-go-v2-service-sts
+  (package
+    (inherit go-github-com-aws-aws-sdk-go-v2)
+    (name "go-github-com-aws-aws-sdk-go-v2-service-sts")
+    (version "1.17.7")
+    (arguments
+     '(#:import-path "github.com/aws/aws-sdk-go-v2/service/sts"
+       #:unpack-path "github.com/aws/aws-sdk-go-v2"))
+    (propagated-inputs
+     (list go-github-com-aws-smithy-go))))
+
 (define-public go-github-com-aws-smithy-go
   (package
     (name "go-github-com-aws-smithy-go")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 0ef1ba4edf..5aedd1d51e 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -11300,16 +11300,6 @@ (define-public go-github-com-dvsekhvalnov-jose2go
     (home-page "https://github.com/dvsekhvalnov/jose2go")
     (license license:expat)))
 
-(define-public go-github-com-aws-aws-sdk-go-v2-service-sts
-  (package
-    (inherit go-github-com-aws-aws-sdk-go-v2)
-    (name "go-github-com-aws-aws-sdk-go-v2-service-sts")
-    (version "1.17.7")
-    (arguments
-     '(#:import-path "github.com/aws/aws-sdk-go-v2/service/sts"
-       #:unpack-path "github.com/aws/aws-sdk-go-v2"))
-    (propagated-inputs (list go-github-com-aws-smithy-go))))
-
 (define-public go-github-com-aws-aws-sdk-go-v2-service-s3
   (package
     (inherit go-github-com-aws-aws-sdk-go-v2)
-- 
2.41.0

